src/QY2Styler.cc

```diff
@@ -142,9 +143,40 @@ bool QY2Styler::loadStyleSheet( const QString & filename )
     }
 }
 
+const QString QY2Styler::buildStyleSheet(QString content)
+{
+    QStringList alreadyImportedFilenames;
+    return buildStyleSheet(content, alreadyImportedFilenames);
+}
+
+const QString QY2Styler::buildStyleSheet(QString content, QStringList & alreadyImportedFilenames)
+{
+    QRegularExpression re(" *@import +url\\(\"(.+)\"\\);");
```

Comment:
It's sad that the Qt CSS parser has no public API, otherwise we could easily use it instead of this simple regexp. See http://stackoverflow.com/a/31593423.